google/ggrc-core
# GGRC-4672 Remove cycle requests on /dashboard page

# Issue description

Opening /dashboard triggers too many requests, causing the whole app to become unusable. We are going to disable all the extra queries for the 1.6.0 release to unblock the users and implement a proper solution in 1.7.0.

# Steps to test the changes

Have a workflow with more than 1000 tasks in the system. Open the dashboard page and observe the requests executed in the network tab in the dev tools.

# Solution description

We disable loading cycle objects and only load all the active workflows assigned to the user. There are a few changes in functionality:

1. Task stats will no longer be shown (number of overdue tasks, completed tasks, total task count, next due date)
2. The order of workflows displayed on the my workflows list will now be different (we used to order by task date, now it's alphabetically by the workflow title).
3. Number of workflows shown in the My Workflows section might now be different (we used to filter out workflows that don't have active tasks, but now we filter based on workflow status so workflows that are active, but don't have an active cycle that were filtered out before will now be shown). 
4. Overdue tasks notification will no longer be shown.

Before:
![screen shot 2018-03-12 at 23 17 21](https://user-images.githubusercontent.com/513444/37316669-5d0f146c-2658-11e8-9fe0-5b6b6b9db7e7.png)

After:
![screen shot 2018-03-13 at 00 07 14](https://user-images.githubusercontent.com/513444/37316674-5f82c892-2658-11e8-9e62-17d0510537f5.png)


# Sanity checklist




# PR Review checklist
